packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -13,17 +13,22 @@ export interface Props {
   data: DataFrame;
   width: number;
   height: number;
+  /** pixels in number */
+  columnMinWidth?: number;
   onCellClick?: TableFilterActionCallback;
 }
 
-export const Table = ({ data, height, onCellClick, width }: Props) => {
+export const Table = ({ data, height, onCellClick, width, columnMinWidth }: Props) => {
```

Comment:
Is this a FunctionComponent? Maybe it should be be typed as that

Reply:
yea, and should use React.memo() to make it not re-render when props are the same